client/landing/gutenboarding/onboarding-block/designs/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { isEnabled } from '@automattic/calypso-config';
 import DesignPicker, {
 	FeaturedPicksButtons,
+	PremiumBadge,
```

Comment:
This change triggers the following TypeScript error:

```javascript
yarn tsc -p client/tsconfig.json --noEmit

client/landing/gutenboarding/onboarding-block/designs/index.tsx:1:24 - error TS2614: Module '"@automattic/design-picker"' has no exported member 'PremiumBadge'. Did you mean to use 'import PremiumBadge from "@automattic/design-picker"' instead?

1 import DesignPicker, { PremiumBadge, getAvailableDesigns } from '@automattic/design-picker';
                         ~~~~~~~~~~~~
```

Could you fix it @arthur791004? Thanks!

Reply:
Well apparently this is an issue with `tsc`, and it's just a matter of rebuilding packages to make those errors go away (more information [here](https://github.com/Automattic/wp-calypso/pull/60274#discussion_r790978873)). Sorry for the ping Arthur.